TransbankPosSDK/Responses/LastSaleResponse.cs

```diff
@@ -11,7 +11,7 @@ public class LastSaleResponse : LoadKeysResponse
         {
             { "Ticket", 4},
             { "AutorizationCode", 5},
-            { "Ammount", 6},
```

Comment:
In order to retain backwards compatibility, you will need to create a property called `Ammount` who sets and return te value of `Amount`

Reply:
@gonzunigad let's not forget about this fix.